Sources/ProjectDescription/PluginLocation.swift

```diff
@@ -15,24 +15,24 @@ public struct PluginLocation: Codable, Equatable {
         PluginLocation(type: .local(path: path))
     }
 
-    /// A `URL` to a `git` repository pointing at a `tag`.
+    /// A `URL` to a `git` repository pointing at a `tag`, and optionally a subfolder.
```

Comment:
Let's also revise the documentation and add a reasoning why this would be useful.